api/rest/client/lbclient.go

```diff
@@ -13,27 +12,44 @@ import (
 	peer "github.com/libp2p/go-libp2p-peer"
 )
 
+// Strategy values
+const (
+	Random Strategy = iota
+	RoundRobin
+)
+
+// Strategy represents load balancing strategy
+type Strategy int
+
 // loadBalancingClient a client to interact with IPFS Cluster APIs.
 // It balances the load by distributing it among peers.
 type loadBalancingClient struct {
-	clients           []*Client
-	shedulingStrategy string
+	clients  []*Client
```

Comment:
No need to make a pointer to an interface, you can use `Client` directly (the underlying type is probably already a pointer).